lib/middleware/validate-request-query-middleware.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+/**
+ * Parse an incoming request and validate that the query matches a schema.
+ *
+ * Use this as middleware in your route files.
+ */
```

Comment:
Yes.  File descriptions (this type of comment) come before "use strict".  I've updated all middleware files.